db/schema.rb

```diff
@@ -476,8 +476,9 @@
     t.string   "muuttaja",   limit: 50,                           default: "",  null: false
   end
 
-  add_index "budjetti_tuote", ["yhtio", "kausi", "tuoteno", "osasto", "try"], name: "tubu", unique: true, using: :btree
-  add_index "budjetti_tuote", ["yhtio", "tuoteno", "kausi"], name: "yhtio_tuote_kausi", using: :btree
+  # Specified key was too long; max key length is 1000 bytes
+  #add_index "budjetti_tuote", ["yhtio", "kausi", "tuoteno", "osasto", "try"], name: "tubu", unique: true, using: :btree
+  #add_index "budjetti_tuote", ["yhtio", "tuoteno", "kausi"], name: "yhtio_tuote_kausi", using: :btree
```

Comment:
Specified key was too long; max key length is 1000 bytes. Mitäs täl pitäis tehä?